### PR TITLE
myal-7264 taxes and fees for b2u channels

### PR DESCRIPTION
**Pull request by roman8cherniavsky**

### CODE REVIEW

File: examples/methods/UpdateTaxes.yaml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+      verb: UpdateTaxes
+      ota_property_id: lake_view_hotel
+      ota_property_password: very-secret-password
+      ota_property_sub_id: '10034818'
+      mya_property_id: 25678
+      guid: 6C08B96E-450D-4E6A-9933-7D0305730305
+      shared_secret : s3cr3ts4uc3
+      ota_cid: ota
+      taxes:
+        update:
```
Comment (roman8cherniavsky) [inline examples/methods/UpdateTaxes.yaml:15]: When I do `$ref: ../taxes.yaml` it renders to 
ref_not_rendered
But it starts working when I delete `$ref: ../taxes.yaml` from `CreateProperty` example.
So I decided to put content of `../taxes.yaml` here.

File: methods/UpdateTaxes.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            - type: object
+              properties:
+                taxes:
+                  type: object
+                  description: Object that specifies Taxes/Fees to update and/or delete.
+                  properties:
+                    update:
+                      type: array
+                      description: >
+                        List of Taxes/Fees that were updated or created
```
Comment (cloudbeds-mo) [inline methods/UpdateTaxes.yaml]: List of taxes/fees to update or create

File: methods/UpdateTaxes.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                      items:
+                        $ref: ../schemas.d/tax_info.yaml
+                    delete:
+                      type: array
+                      description: >
+                        List of Taxes/Fees that were deleted
```
Comment (cloudbeds-mo) [inline methods/UpdateTaxes.yaml]: List of taxes/fees to delete

File: methods/UpdateTaxes.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                      items:
+                        type: object
+                        description: Tax/Fee object to delete. Contains ID only.
+                        properties:
+                          id:
+                            type: integer
+                            minimum: 1
+                            description: Tax/Fee ID in cloudbeds system
```
Comment (cloudbeds-mo) [inline methods/UpdateTaxes.yaml]: Tax/fee ID in Cloudbeds' system

File: schemas.d/tax_conditions.yaml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+description: >
+  Array of conditions that determine if this amount will be applied.
+items:
+  type: object
+  properties:
+    min_start:
+      type: string
+      description: >
+        Amount of parameter above which tax is applied. For example,
+        if condition_parameter is "total-price" and here we have 200,
```
Comment (cloudbeds-mo) [inline schemas.d/tax_conditions.yaml]: use backticks around parameters like `condition_parameter`

File: schemas.d/tax_info.yaml [hunk 6]
```diff
@@ -0,0 +1,125 @@
+---
+type: object
+properties:
+  id:
+    type: integer
+    minimum: 1
+    description: Tax/Fee ID in cloudbeds system
```
Comment (cloudbeds-mo) [inline schemas.d/tax_info.yaml]: Tax/fee ID in Cloudbeds' system

File: schemas.d/tax_info.yaml [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+  is_fee:
+    type: boolean
+    description: '"true" if fee. Otherwise tax'
```
Comment (cloudbeds-mo) [inline schemas.d/tax_info.yaml]:
```
`true`
```
instead of `"true"`

Comment (cloudbeds-mo) [reply schemas.d/tax_info.yaml]:
```
`true` if it's a fee, otherwise it's a tax.
```

File: methods/UpdateTaxes.yaml [hunk 8]
```diff
@@ -0,0 +1,58 @@
+---
+post:
+  operationId: update_taxes
+  description: >
+    Synchronize taxes/fees after they have been changed.
```
Comment (cloudbeds-mo) [inline methods/UpdateTaxes.yaml]: Synchronize taxes/fees after they have been changed on Cloudbeds.

**Review by cloudbeds-mo — APPROVED** [review]

**Pull request merged**